great_expectations/data_context/store/store_backend.py

```diff
@@ -464,42 +406,40 @@ def _convert_key_to_filepath(self, key):
         self._validate_key(key)
 
         converted_string = self.filepath_template.format(*list(key), **{
-            "file_extension" : self.file_extension,
-            "file_prefix" : self.file_prefix,
+            "file_extension": self.file_extension,
+            "file_prefix": self.file_prefix,
         })
 
         return converted_string
 
-
     def _convert_filepath_to_key(self, filepath):
 
         # Convert the template to a regex
         indexed_string_substitutions = re.findall("\{\d+\}", self.filepath_template)
-        tuple_index_list = ["(?P<tuple_index_{0}>.*)".format(i,) for i in range(len(indexed_string_substitutions))]
+        tuple_index_list = ["(?P<tuple_index_{0}>.*)".format(i, ) for i in range(len(indexed_string_substitutions))]
```

Comment:
Out of curiosity, are you using a linter?

Reply:
I am not. We can discuss linters during the offsite. I want to make sure they don't create too many edits :)